.github/workflows/codeception.yml

```diff
@@ -29,7 +29,6 @@ jobs:
         strategy:
             matrix:
                 include:
-                    - { php-version: 8.0, database: "mariadb:10.7", server_version: "10.7.7-MariaDB-1:10.7.7+maria~ubu2004", dependencies: lowest, experimental: false }
```

Comment:
we should still run lowest dependencies tests

Reply:
With php-version 8.1, right ?

Reply:
yes, since this is the min requirement in composer.json now.